### ISSUE TITLE
Make "settings" accessible.

### ISSUE THREAD

**Issue by sabbamitta** [sc-voice/sc-voice#109]
#### User story

As a user depending on a screen reader to navigate the settings, this page (modal) is highly confusing. There are several issues that need to be fixed, a couple highlighted in this video:

https://drive.google.com/open?id=1aIV-Q2sLN2n10pzhU9fUvc6ygb42hnQG

Some key issues: 
- The heading element from the home page 'bleeds' into this page.
- ARIA labels are poorly used, screen readers (a) duplicate some info (b) cannot support the user information about the structure of the page.
- Illogical flow (immediately after opening, users land on "close settings")

This ticket should be considered in relation to the work done on settings associated with #7. 

#### Feature description (the list of things that need to be done for the ticket to be considered finished):

- [ ] TBC

#### Pre release planning check:
- [ ] Small enough to completed in a release.
- [ ] Dependencies marked
- [ ] No external dependencies block this from being completed.
- [ ] Details are understood by dev team so they know what needs to be done and can determine if it can be completed.

**Comment by sabbamitta** [comment]
This issue is merged with #7.
